playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -10,6 +10,9 @@
   remote_user: root
```

Comment:
This entire playbook should probably be killed now that we support configuring s3 support in openshift_hosted for the router.